Documentation/API.md

```diff
@@ -679,6 +679,24 @@ public protocol ExtensionMap {
 }
 ```
 
+## Descriptors
+
+Some other languages expose _Descriptor_ objects for messages, enums, fields,
+and oneof; but not all languages.  And the `.proto` language allows developers
```

Comment:
s/;/,/

s/And the \`.proto\` language/The \`.proto\` language also/

Reply:
Done